action.yml

```diff
@@ -0,0 +1,163 @@
+name: 'deploy-infrastructure'
+author: '1up-team'
+description: 'Action to plan and deploy Terraform modules'
+
+inputs:
+  module-name:
+    description: 'Module name, usually equal to its directory name or service name'
+    required: false
+    default: 'terraform'
+  module-path:
```

Comment:
terraform-folder-path or terraform-stack-folder-path would probably make more sense as this is a terraform folder path, and it shouldn't contain / in front; examples could have been: terraform, terraform/vdfproxy-foobar